transport/src/main/java/io/netty/channel/AbstractServerChannel.java

```diff
@@ -58,6 +60,13 @@ protected void doDisconnect() throws Exception {
         throw new UnsupportedOperationException();
     }
 
+    @UnstableApi
+    @Override
+    protected final void doShutdownOutput(Throwable cause) throws Exception {
+        super.doShutdownOutput(cause);
```

Comment:
Should this be wrapped in try finally since do shutdown throws exception?

Reply:
I don't think this is necessary because if this method throws AbstractChannel will catch and close the connection.

Reply:
@Scottmitch ah ok... sounds good then